src/lib.rs

```diff
@@ -225,15 +225,22 @@ struct Lighthouse {
 #[pymethods]
 impl Lighthouse {
     #[new]
-    fn new(py: Python<'_>, bind: String, min_replicas: u64) -> PyResult<Self> {
+    fn new(
+        py: Python<'_>,
+        bind: String,
+        min_replicas: u64,
+        join_timeout_ms: Option<u64>,
+    ) -> PyResult<Self> {
+        let join_timeout_ms = join_timeout_ms.unwrap_or(100);
+
         py.allow_threads(move || {
             let rt = Runtime::new()?;
 
             let lighthouse = rt
                 .block_on(lighthouse::Lighthouse::new(lighthouse::LighthouseOpt {
                     bind: bind,
                     min_replicas: min_replicas,
-                    join_timeout_ms: 100,
+                    join_timeout_ms: join_timeout_ms,
                     quorum_tick_ms: 100,
```

Comment:
While here, do we also want to make `quorum_tick_ms` configurable?
Also, if possible, can we add a small unit test. 
Can be done in a separate PR or TODO for someone :-)

Reply:
Thanks for the suggestion. Implemented :)
Not sure how to test quorum_tick though, seems like this would be quite flaky